### PR TITLE
[JN-1294] In-person kit i18n

### PR DESCRIPTION
**Pull request by MatthewBemis**
#### DESCRIPTION (include screenshots, and mobile screenshots for participant UX)

Adds i18n support for in-person kits.

Also includes a few text changes, per Parker's feedback

#### TO TEST:  *(simple manual steps for confirming core behavior -- used for pre-release checks)*

Go through the in-person kits flow in any language and confirm that all of the new texts display correctly

### CODE REVIEW

File: ui-admin/src/study/kits/KitEnrolleeSelection.tsx [hunk 1]
```diff
@@ -188,9 +188,9 @@ export default function KitEnrolleeSelection({ studyEnvContext }: { studyEnvCont
         <RowVisibilityCount table={table}/>
       </div>
       <div className="d-flex">
-        { user?.superuser && <Link to={'../scan'}>
+        <Link to={'../scan'}>
```
Comment (MatthewBemis) [inline ui-admin/src/study/kits/KitEnrolleeSelection.tsx:189]: Driveby fix to remove Scan kit button from behind a superuser flag 🤦

**Review by connorlbark — APPROVED** [review]

File: populate/src/main/resources/seed/i18n/dev/languageTexts.json [hunk 2]
```diff
@@ -101,5 +101,39 @@
...
+  {"keyName": "kitsPageInPersonCompleteButton", "text": "DEV_Complete a kit in-person", "language": "dev"},
+  {"keyName": "kitsPageYourKitsTitle", "text": "DEV_Your kits", "language": "dev"},
+  {"keyName": "kitsPageStatusPreparing", "text": "DEV_Preparing", "language": "dev"},
+  {"keyName": "kitsPageStatusShipped", "text": "DEV_Shipped", "language": "dev"},
+  {"keyName": "kitsPageStatusReturned", "text": "DEV_Returned", "language": "dev"},
+  {"keyName": "kitsPageStatusCreated", "text": "DEV_Created", "language": "dev"},
+  {"keyName": "kitsPageStatusCollected", "text": "DEV_Collected", "language": "dev"},
+  {"keyName": "kitsPageNoKits", "text": "DEV_You do not have any sample collection kits at this time.", "language": "dev"},
+  {"keyName": "inPersonKitsTitle", "text": "DEV_Sample kit instructions", "language": "dev"},
+  {"keyName": "inPersonKitsDescription", "text": "DEV_If you are completing a sample collection kit in-person, please follow the instructions provided by a member of the study team. Any additional information that you may need, such as your unique participant identifier, will be provided below.", "language": "dev"},
```
Comment (devonbush) [inline populate/src/main/resources/seed/i18n/dev/languageTexts.json]: I would lean towards having these keys be "kitsInPerson..." just so they'll be grouped alphabetically with other kit stuff

**Review by devonbush — APPROVED** [review]
